File: android/app/src/main/java/com/TMR24/MotionStudy/PositionSettingObjectMap.java
package com.TMR24.MotionStudy;

import java.util.Map;

public class PositionSettingObjectMap {
    public int idButton;
    public String idSettingsButton;
    public Map <String, Object> dataSettingsButton;

    public PositionSettingObjectMap(int idButton, String idSettingsButton, Object dataSettingsButton) {
        this.idButton = idButton;
        this.idSettingsButton = idSettingsButton;
        this.dataSettingsButton = (Map < String, Object >) dataSettingsButton;
    }
}

File: android/app/src/main/java/com/TMR24/MotionStudy/PositionSettingObjectMap.kt
package com.TMR24.MotionStudy

class PositionSettingObjectMap(var idButton: Int, var idSettingsButton: String, dataSettingsButton: Any?) {
    var dataSettingsButton: Map<String, Any>?

    init {
        this.dataSettingsButton = dataSettingsButton as Map<String, Any>?
    }
}